### PR TITLE
Create Coordinate Picker plugin

### PR DESCRIPTION
**Pull request by jatowler**
Listens for mouse clicks on the map, transforms them to a specified
frame, and prints a list of them in the plugin GUI. The most recent
coordinates are copied to the clipboard.

### CODE REVIEW

File: mapviz_plugins/src/coordinate_picker_plugin.cpp [hunk 1]
```diff
@@ -0,0 +1,223 @@
+// *****************************************************************************
+//
+// Copyright (c) 2016, Southwest Research Institute® (SwRI®)
```
Comment (malban) [inline mapviz_plugins/src/coordinate_picker_plugin.cpp]: might as well make the date 2018

**Comment by agyoungs** [comment]
For consistency with the rest of mapviz, can the order of the latitude and longitude coordinates be swapped for the ws84 frame?  Currently the coordiate picker spits it out in (lon, lat) ordering.  It appears the rest of mapviz shows (lat, lon) ordering.

**Comment by jatowler** [comment]
The picker uses (lon, lat) because it works like (x,y). But it’s easily changed if others agree.

**Comment by pjreed** [comment]
If it displays (lat, lon) everywhere else, I'd probably rather use that, even though I always get confused when something stuffs WGS84 coordinates into a data structure with an "x" and a "y" and I have to remember that in that case it's (lon, lat).

**Comment by agyoungs** [comment]
The only other thing that I noticed that might be nice is a clear button to clear the coordinates.  That's not a big deal to me though.

**Comment by jatowler** [comment]
It turns out the order of display is defined by the order returned from `TransformManager`. I can add an exception for `wgs84` to reverse the order, but that seems fragile. Is that the right way to go about it?

**Pull request merged**